segmented_sieve_of_eratosthenes/segmentedSieveOfEratosthenes.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+#include <cmath>
+#include <cstring>
+#include <vector>
+using namespace std;
+
+vector<int> Prime; //contains prime numbers uptop segMax
+void segmentedSieve(){
+	#define segMax 1000000 //till point you want to find Primes. Preferebly pow(10,x)
```

Comment:
Better take this as a parameter to the function, instead of hardcoding it.

Reply:
sure will change it